OPERATION.md

```diff
@@ -245,7 +245,7 @@ To access the Menu Level 2, use the `Top Dial` on the top of the wand to move do
 | Option | Purpose | Intensify Button | Barrel Wing Button |
 |:------:|---------|------------------|--------------------|
 | 5 | Firing&nbsp;Mode&nbsp;Selection | Cycle through VG/CTS operation modes<sup>4</sup> | Cycle through VG colour modes <sup>5</sup> |
-| 4 | Smoke/Overheat&nbsp;Toggles | Enable/Disable Smoke Effects | Enable/Disable Overheating |
+| 4 | Overheating / Smoke | Enable/Disable Overheating Ability | Enable/Disable Smoke Effects |

```

Comment:
Curious as to why this was simply a swap of the options? Does it feel more logical this way?